tests/modules/nf-core/universc/count/test.yml

```diff
@@ -0,0 +1,42 @@
+- name: universc count test_universc_count
+  command: nextflow run ./tests/modules/nf-core/universc/count -entry test_universc_count -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/universc/count/nextflow.config
+  tags:
+    - universc
+    - universc/count
+  files:
+    - path: output/universc/genome.filtered.gtf
+      md5sum: 9dc1a510bbe4438cfc2a74423bf2b2bd
+    - path: output/universc/homo_sapiens_chr22_reference/fasta/genome.fa
+      md5sum: f315020d899597c1b57e5fe9f60f4c3e
+    - path: output/universc/homo_sapiens_chr22_reference/fasta/genome.fa.fai
+      md5sum: 3520cd30e1b100e55f578db9c855f685
+    - path: output/universc/homo_sapiens_chr22_reference/reference.json
+      md5sum: 1f7bf05454cc908bf70cd232ae70b02d
+    - path: output/universc/homo_sapiens_chr22_reference/star/Genome
+      md5sum: 22102926fadf5890e905ca71b2da3f35
+    - path: output/universc/homo_sapiens_chr22_reference/star/SA
+      md5sum: bcf3e1a855783105150b46c905465333
+    - path: output/universc/homo_sapiens_chr22_reference/star/SAindex
+      md5sum: b93fb07d342e6c32a00ebc4311c0ad38
+    - path: output/universc/homo_sapiens_chr22_reference/star/chrLength.txt
+      md5sum: c81f40f27e72606d7d07097c1d56a5b5
+    - path: output/universc/homo_sapiens_chr22_reference/star/chrName.txt
+      md5sum: 5ae68a67b70976ee95342a7451cb5af1
+    - path: output/universc/homo_sapiens_chr22_reference/star/chrNameLength.txt
+      md5sum: b190587cae0531f3cf25552d8aa674db
+    - path: output/universc/homo_sapiens_chr22_reference/star/chrStart.txt
+      md5sum: bc73df776dd3d5bb9cfcbcba60880519
+    - path: output/universc/homo_sapiens_chr22_reference/star/exonGeTrInfo.tab
+      md5sum: d04497f69d6ef889efd4d34fe63edcc4
+    - path: output/universc/homo_sapiens_chr22_reference/star/exonInfo.tab
+      md5sum: 0d560290fab688b7268d88d5494bf9fe
+    - path: output/universc/homo_sapiens_chr22_reference/star/geneInfo.tab
+      md5sum: a20c70b081f5d83649c48ebbd951cb77
+    - path: output/universc/homo_sapiens_chr22_reference/star/genomeParameters.txt
+      contains: ["genomeGenerate"]
+    - path: output/universc/homo_sapiens_chr22_reference/star/transcriptInfo.tab
+      md5sum: 6fa11b4d34f4680a1c23dbcea2e050d5
+    - path: output/universc/sample-123/outs/metrics_summary.csv
+      md5sum: edfa1a0dc666c38f9740167c045fb3c8
+    - path: output/universc/sample-123/outs/web_summary.html
```

Comment:
Yeah that one is still not passing, should ditch md5 and just search for an exemplary output line as for others using `contain` if this is possible.